doc/concepts/embeddings.rst

```diff
@@ -0,0 +1,45 @@
+.. _embeddings:
```

Comment:
The section reads very nice! 

There is one thing that might confuse readers (but which you might not know), the embedding methods take a single feature vector, not an entire data set. In QML, the data embedding is often useful, but for variational circuits the standard situation embedds one input at a time (like in neural nets). 

I suggest just pointing this out clearly, for example by saying "this is how 1 vector gets encoded like PL does it", and then "in principle, this can be used to encode an entire dataset, how NOT done by PL" :)

Reply:
In all other sections in concepts, only the theoretical/conceptual descriptions are given. They do not talk about how it was exactly implemented in PennyLane and give no links to the corresponding implemented functions. 

I can mention it in the tutorials/advanced usage section. 

What do you think?

Reply:
> In all other sections in concepts, only the theoretical/conceptual descriptions are given. They do not talk about how it was exactly implemented in PennyLane and give no links to the corresponding implemented functions.

This is true, and something we should perhaps keep consistent. Maybe, instead of elaborating on the code/PennyLane specifics, we should start adding `.. see-also::`'s to the relevant tutorials?

Reply:
Good idea

Reply:
I will change it for all concept tutorials.